src/Servers/IIS/IntegrationTesting.IIS/src/IISExpressDeployer.cs

```diff
@@ -453,49 +453,91 @@ private class WindowsNativeMethods
             internal static extern bool EnumWindows(EnumWindowProc callback, IntPtr lParam);
             [DllImport("user32.dll", SetLastError = true, CharSet = CharSet.Auto)]
             internal static extern int GetClassName(IntPtr hWnd, StringBuilder lpClassName,int nMaxCount);
+            [DllImport("kernel32.dll")]
+            internal static extern IntPtr CreateRemoteThread(IntPtr hProcess, IntPtr lpThreadAttributes, uint dwStackSize, IntPtr lpStartAddress, IntPtr lpParameter, uint dwCreationFlags, IntPtr lpThreadId);
+
         }
 
         private void SendStopMessageToProcess(int pid)
         {
-            Logger.LogInformation($"Sending shutdown request to {pid}");
             var found = false;
-            WindowsNativeMethods.EnumWindows((ptr, param) => {
-                WindowsNativeMethods.GetWindowThreadProcessId(ptr, out var windowProcessId);
-                if (pid == windowProcessId)
-                {
-                    // 256 is the max length
-                    var className = new StringBuilder(256);
+            var extraLogging = false;
```

Comment:
Seems odd to have this bool to avoid one log call.

Reply:
It's a log call that logs a lot.